Sample App/Fattmerchant-iOS-SDK-Sample/ViewController.swift

```diff
@@ -27,7 +27,25 @@ class ViewController: UIViewController, TransactionUpdateDelegate, MobileReaderC
   }
 
   @IBAction func onConnectReaderButtonPress(_ sender: UIButton) {
+    #if targetEnvironment(simulator)
+    let alertController = UIAlertController(title: "Number of readers wanted", message: "", preferredStyle: UIAlertController.Style.alert)
+    alertController.addTextField { (textField : UITextField!) -> Void in
+      textField.placeholder = "Enter number of readers needed"
+    }
+    let saveAction = UIAlertAction(title: "Save", style: UIAlertAction.Style.default, handler: { alert -> Void in
+      let firstTextField = alertController.textFields![0] as UITextField
```

Comment:
using an assertion here is risky... we can use other safe rapping like if let or guard